### ISSUE TITLE
Export email to EML format- Option to set read-only file attribute

### ISSUE THREAD

**Issue by josepbb** [thundernest/import-export-tools-ng#164]
There is a well know windows issue related to file-modified date (time-stamp) in EML files.
When a message is exported as EML from ImportExportTools, the file-modified date is set to the message date.
Then, in Windows Explorer, if you either open or over on this EML file, the file-modified date is set to the current windows time.
This could be fixed, if ImportExportTools NG sets the read-only attribute in the file.
To prevent either compatibility or side issues, it could be added as an option in the General Parameters.

**Comment by cleidigh** [comment]
@josepbb 
you are correct, this is a weird Windows issue I have looked at a bit in the past.
I am concerned it's a bit of a hack with unintentional side effects.
probably would be to make this some form of option with a warning
Christopher
